### PR TITLE
#32: implemented Openshift CLI toolCommandlet

### PR DESCRIPTION
**Pull request by salimbouch**
related issue: #32

### CODE REVIEW

**Review by hohwille — APPROVED** [review]
@salimbouch thanks for your PR. It seems that `oc` is really simple with no bells & whistles so all correct. 👍 
Ready for merge but could you have a look at the encoding issue?

File: cli/src/main/resources/nls/Ide_de.properties [hunk 1]
```diff
@@ -6,6 +6,7 @@ cmd-env=Gibt die zu setztenden und exportierenden Umgebungsvariablen aus.
 cmd-get-version=Zeigt die Version des selektierten Werkzeugs an.
 cmd-gh=Werkzeug Kommando für die Github Kommandoschnittstelle.
 cmd-helm=Werkzeug Kommando für Helm (Kubernetes Package Manager)
+cmd-oc=Werkzeug Kommando f�r Openshift CLI (Kubernetes Management Tool)
```
[the remark this answers is not in the document]
Comment (hohwille) [reply cli/src/main/resources/nls/Ide_de.properties:9]: I have reviewed PR #28 that did not have this problem. Therefore, I assume that something went wrong with your encoding when you did edit this file. Please check and fix in both PRs. Then we can merge. Thanks in advance.

[the remark this answers is not in the document]
Comment (salimbouch) [reply cli/src/main/resources/nls/Ide_de.properties]: I checked and it looks like my Intellij had the wrong encoding for properties files, I fixed it now.

[the remark this answers is not in the document]
Comment (hohwille) [reply cli/src/main/resources/nls/Ide_de.properties]: Thanks!

**Pull request merged**